<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layer</class>
 <widget class="QWidget" name="layer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_layer_num">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Layer 1</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Thickness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_thichness">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_fit_thick">
         <property name="text">
          <string>Fit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>min:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_thichness_min">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>max:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_thichness_max">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_functional_layer">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>functional</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Roughness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_roughness">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_fit_roug">
         <property name="text">
          <string>Fit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>min:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_roughness_min">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>max:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_roughness_max">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="dsb_material_0">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="l_fraction_0">
         <property name="text">
          <string>Fraction 1:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="l_material_1">
         <property name="text">
          <string>Material 2:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QComboBox" name="cb_material_1">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QComboBox" name="cb_material_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QDoubleSpinBox" name="dsb_material_1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>0.990000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="cb_material_0">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="l_fraction_1">
         <property name="text">
          <string>Fraction 2:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="l_material_0">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Material 1:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="l_material_2">
         <property name="text">
          <string>Material 3:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Composition:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QLabel" name="l_fraction_2">
         <property name="text">
          <string>Fraction 3:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="11">
        <widget class="QDoubleSpinBox" name="dsb_material_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>0.980000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2">
        <widget class="Line" name="lin_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="7" rowspan="2">
        <widget class="Line" name="lin_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="12">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chk_fit_comp">
         <property name="text">
          <string>Fit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X-ray susceptibility:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_auto_x0">
         <property name="text">
          <string>Auto</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bg_x0</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_specify_x0">
         <property name="text">
          <string>Spesify:</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bg_x0</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_x0">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_fit_xrs">
         <property name="text">
          <string>Fit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>min:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_x0_min">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>max:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_x0_max">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.500000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>DW-like factor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_auto_w0">
         <property name="text">
          <string>Auto</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bg_w0</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_specify_w0">
         <property name="text">
          <string>Spesify:</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bg_w0</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_w0">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_fit_dwlf">
         <property name="text">
          <string>Fit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>min:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_w0_min">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>max:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_w0_max">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.500000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pb_add_above">
       <property name="text">
        <string>Add layer above</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_move_up">
       <property name="text">
        <string>Move up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_delete">
       <property name="text">
        <string>Delete layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_move_down">
       <property name="text">
        <string>Move down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_add_below">
       <property name="text">
        <string>Add layer below</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>dsb_w0_max</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>dsb_w0_min</zorder>
  <zorder>sb_thichness_max</zorder>
  <zorder>sb_thichness_min</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>dsb_roughness_min</zorder>
  <zorder>dsb_roughness_max</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>line_4</zorder>
  <zorder>line_4</zorder>
  <zorder>line_3</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_x0"/>
  <buttongroup name="bg_w0"/>
 </buttongroups>
</ui>
